lib/v2/vcb/index.js

```diff
@@ -0,0 +1,58 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { cat } = ctx.params;
+
+    const baseUrl = 'https://vcb-s.com';
+    const currentUrl = cat ? `${baseUrl}/archives/category/${cat}` : `${baseUrl}/`;
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+
+    const articleLinks = $('div.title-article>h1>a')
+        .toArray()
+        .map((a) => $(a).attr('href'));
+
+    const items = await Promise.all(
+        articleLinks.map((link) =>
+            ctx.cache.tryGet(link, async () => {
+                const detailResponse = await got({
+                    method: 'get',
+                    url: link,
+                });
+
+                const $$ = cheerio.load(detailResponse.data);
+
+                const title = $$('div.title-article>h1>a').text().trim();
+                const pubDate = $$('div.tag-article>span').eq(0).text().trim();
+                const author = $$('div.tag-article a[rel="author"]').text().trim();
+                const content = $$('div.centent-article')
+                    .html()
+                    .replace(/<div.+?abh_box[\w\W]+/g, '')
+                    .replace(/<p.+?medie-info-switch[\w\W]+/g, '')
+                    .replace(/<div.+?dw-box-download.+?>([\w\W]+?)<\/div>/g, '<pre>$1</pre>');
+
+                return {
+                    title,
+                    link,
+                    description: content,
+                    author,
+                    pubDate: timezone(parseDate(`20${pubDate}`, 'YYYY-M-D'), +8),
+                };
+            })
+        )
+    );
```

Comment:
Since this site is a WordPress based, you are suggested to use WordPress API instead of going through every pages.

Offical docs: https://developer.wordpress.org/rest-api/key-concepts/

All API endpoint: `/wp-json/wp/v2`
Get posts: `/wp-json/wp/v2?per_page=100`
Get posts under specific category: `/wp-json/wp/v2/posts?categories=${id}&per_page=100`
Get categories: `/wp-json/wp/v2/categories`

Reply:
latest commit has use WordPress api